vispy/gloo/buffer.py

```diff
@@ -299,9 +299,9 @@ def __setitem__(self, key, data):
         elif data.size > stop - start:
             raise ValueError('Data too big to fit GPU data '
                              '(%d > %d-%d).' % (data.size, stop, start))
-        
+
         # Set data
-        offset = start  # * self.itemsize
+        offset = start
```

Comment:
So `start` is in item-space, right? It is then passed to `set_subdata` as `offset`. You then updated the `set_subdata` method to say that `offset` is in "bytes", right? Or is `start` in number of bytes (if so where does that happen)?

Reply:
Here, yes,  `start` is in item space. It is passed below to `DataBuffer.set_subdata()`, which takes `offset` in item space, converts it to byte space and forwards it `Buffer.set_subdata()`.
Note the difference between space of `offset` in `DataBuffer.set_subdata()` and `Buffer.set_subdata()`.
I did not touch the code, just the doc.
The comment at the line above maybe comes from this confusion.

Reply:
OMG :man_facepalming: 

I read your change to the docstring as you *adding* the `in bytes`. I now see you removed it. Thank you for your patience.

Reply:
No worry. You are welcome!